content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -268,7 +268,7 @@ For Calico to work correctly, you need to pass `--pod-network-cidr=192.168.0.0/1
 
 ```shell
 kubectl apply -f https://docs.projectcalico.org/v3.3/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
```

Comment:
to me it seems that this line is no longer needed.
the 3.4 yaml includes ClusterRoleBinding and ClusterRole objects.